com.unity.shadergraph/Editor/Data/Nodes/Input/Texture/ProceduralVirtualTextureNode.cs

```diff
@@ -0,0 +1,105 @@
+using System.Collections.Generic;
+using UnityEditor.ShaderGraph.Drawing.Controls;
+using UnityEngine;
+using UnityEditor.Graphing;
+using UnityEditor.ShaderGraph.Internal;
+
+namespace UnityEditor.ShaderGraph
+{
+#if PROCEDURAL_VT_IN_GRAPH
+    [Title("Input", "Texture", "Procedural Virtual Texture")]
+    class ProceduralVirtualTextureNode : AbstractMaterialNode
```

Comment:
Is it now a node act as an input and we have other nodes that do VT sampling?

Reply:
Yes.   Streaming VT is a property, Procedural VT is a node.   Both plug into the SampleVT node to sample from them.